app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -50,12 +53,12 @@ const Form = ({ onClose, refetch, note, isEdit }) => {
             <Button
               className="mr-3"
               disabled={isSubmitting}
-              label={isEdit ? "Update" : "Save changes"}
+              label={isEdit ? t("common.update") : t("common.saveChanges")}
               loading={isSubmitting}
               style="primary"
               type="submit"
             />
-            <Button label="Cancel" style="text" onClick={onClose} />
+            <Button label={t("common.cancel")} style="text" onClick={onClose} />
```

Comment:
Could you tell me why `type="reset"` not passed in the cancel button?

Reply:
The `Form` is yet to be implemented as part of #4.